looprpc/client.proto

```diff
@@ -372,4 +372,11 @@ message QuoteResponse {
     An estimate of the on-chain fee that needs to be paid to sweep the HTLC.
     */
     int64 miner_fee = 3;
+    
+    /**
+    If an error happens during on-chain fee estimation in lnd, this contains
+    the error. The client should decide if it's ok to continue (i.e. external
+    loop in) or not.
+    */
+    string miner_fee_estimation_err = 4;
```

Comment:
Can't we just return a regular error here using the existing gRPC error codes?

Reply:
IIRC, the original issue failed just as a result of an attempted swap execution, or is it that the issue also bled over into the quote call?